aten/src/ATen/native/native_functions.yaml

```diff
@@ -13488,220 +13488,220 @@
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: _fw_primal_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: _make_dual_copy(Tensor primal, Tensor tangent, int level) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: _make_dual_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: view_as_real_copy(Tensor self) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: view_as_real_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: view_as_complex_copy(Tensor self) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: view_as_complex_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: _conj_copy(Tensor self) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: _conj_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
```

Comment:
Not all *_copy should be marked as core. 
Many of these should have been decomposed.